### PR TITLE
Create ci.yml

### PR DESCRIPTION
**Pull request by froi**
Initial commit for GitHub Action for CI using Percy. This would remove the need to use CircleCI.

@eluciano11 we need to add a Percy token to this [repo's secrets](https://github.com/fullstacknights/website/settings/secrets).

### CODE REVIEW

**Comment by eluciano11** [comment]
interesting

Interesting... will add.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: CI
+on: [push, pull_request]
```
Comment (eluciano11) [inline .github/workflows/ci.yml]: I think it would be better to run the CI only on the PR's

**Comment by eluciano11** [comment]
Done!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (froi) [reply .github/workflows/ci.yml]: If we do it that way I don't the the tests trigger when adding fixes to a PR. Does it?

We can test and see if you'd like.

**Comment by froi** [comment]
@eluciano11 I think we're good to go with this one

image

Who ever set up the CircleCI builds needs to turn them off for this project.

**Comment by eluciano11** [comment]
@froi I added Circle CI. I'll turn it off.

**Comment by eluciano11** [comment]
@froi done.

**Comment by eluciano11** [comment]
Beautiful, it worked!

**Pull request merged**